crypto/kyber/pqcrystals-kyber_kyber768_ref/cbd.c

```diff
@@ -0,0 +1,128 @@
+#include <stdint.h>
+#include "params.h"
+#include "cbd.h"
+
+/*************************************************
+* Name:        load32_littleendian
+*
+* Description: load 4 bytes into a 32-bit integer
+*              in little-endian order
+*
+* Arguments:   - const uint8_t *x: pointer to input byte array
+*
+* Returns 32-bit unsigned integer loaded from x
+**************************************************/
+static uint32_t load32_littleendian(const uint8_t x[4])
```

Comment:
All this code duplication :(

Reply:
We should make an effort to have one set of BE/LE functions used throughout... who wants to make a ticket?